pkg/test/framework/features/features.yaml

```diff
@@ -18,7 +18,18 @@ features:
   # features that relate to measuring or exploring your service mesh or the services which comprise it.
   observability:
     telemetry:
-    tracing:
+      stats:
+        prometheus:
+          merge:
+          http:
+            nullvm:
+            wasm:
+          tcp:
+      attributegen:
+      tracing:
+        client-tracing:
```

Comment:
should we have tracing.client-tracing or tracing.client? Not sure which makes more sense @therealmitchconnors

Reply:
Ah, yes, there is clear a dup in naming. I remove `-tracing` from the leaf names.